fail2ban/tests/actiontestcase.py

```diff
@@ -71,6 +71,12 @@ def testSubstituteRecursiveTags(self):
 									'ABC': '123 192.0.2.0',
 									'xyz': '890 123 192.0.2.0',
 								})
+		# obscure embedded case
+		self.assertEqual(CommandAction.substituteRecursiveTags({'A': '<<PREF>HOST>', 'PREF': 'IPV4'}),
+						 {'A': '<IPV4HOST>', 'PREF': 'IPV4'})
```

Comment:
but how you will differentiate, where it should stop - such constructs are very dangerous:
- I need `A` recursively updated to 1.2.3.4 and IPV4HOST = 1.2.3.4
- or possible I need A exact <IPV4HOST> and only IPV4HOST = 1.2.3.4
Some escape? or what?